apps/explorer/src/hooks/useIsMobile.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { BREAK_POINTS, useMediaQuery } from '~/hooks/useMediaQuery';
+
+export function useIsMobile() {
```

Comment:
Instead of this, I'd just something more like `useBreakpoint` and then have that take in a `BREAKPOINT`.